packages/editor-ui/src/api/ctas.ts

```diff
@@ -0,0 +1,12 @@
+import type { IRestApiContext } from '@/Interface';
+import { get } from '@/utils/apiUtils';
+
+export type UserCtas = {
+	becomeCreator: boolean;
+};
+
+export async function getUserCtas(context: IRestApiContext): Promise<UserCtas> {
+	const response = await get(context.baseUrl, '/cta');
+
+	return response;
```

Comment:
Let's return directly?

Reply:
See comment about `return` and `await` above